src/plugins/header/header.ts

```diff
@@ -7,8 +8,25 @@ import {UIPPlugin} from '../../core/base/plugin';
 export class UIPHeader extends UIPPlugin {
   static is = 'uip-header';
 
-  // TODO: note about auto content
-  // option 0 no need of header - do not add a header tag
-  // option 1 user wants default - add a header tag witout child elements
-  // option 2 user wants custom header- user add whatever needed, header skip auto-fill
+  protected connectedCallback(): void {
+    super.connectedCallback();
+    !this.childElementCount && this.autofill();
```

Comment:
I just realized, that in my mind there were direct check of presence of `uip-snippets` and `uip-options`... in case, f.e. customers put smth in there. Does it make any sense, what do you think?

Reply:
Good point!